Cargo.toml

```diff
@@ -26,3 +26,6 @@ log = "~0.3"
 termion = "~1"
 chan = "~0.1"
 chan-signal = "~0.3"
+tempfile = "3"
+which = "2"
+sh_escape = { path = "./pkg/sh_escape" }
```

Comment:
You won't be able to publish the crate to crates.io without specifying a version here.

Reply:
Ah, darn. Thanks.

I guess I'll have to think of an okay name for it so I can publish it separately then.